test/dotnet.Tests/GivenThatIWantToManageMulticoreJIT.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.Extensions.PlatformAbstractions;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+using Xunit.Abstractions;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.Tests
+{
+    public class GivenThatIWantToManageMulticoreJIT : TestBase
+    {
+        ITestOutputHelper _output;
+        private const string OptimizationProfileFileName = "dotnet";
+        
+        public GivenThatIWantToManageMulticoreJIT(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public void When_invoked_then_dotnet_writes_optimization_data_to_the_profile_root()
```

Comment:
As we clean up tests we're moving them to snake_casing and When_Then names both to capture tested intent and to make that intent easier to parse during test report read.